File: week01/pset1/02-mario/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

void print_row(int bricks);
void print_col(int height);

int main(void){
    int height;

    do{
        height = get_int("Height: ");
    } while(height < 1 || height > 8);

    print_col(height);

}

void print_row(int spaces, int bricks)
{
    for(int i = 0; i < bricks; i++)
    {
        printf("#");
    }
    printf("\n");
}

void print_col(int height){
    for(int i = 0; i < height; i++){
        int space = height - i - 1;
        print_row(i + 1);
    }
}

File: week06/pset6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    n = get_int("Height: ")
    if 1 <= n <= 8:
        break

for i in range(1, n + 1):
    spaces = n - i
    hashes = i

    print(" " * spaces, end="")
    print("#" * hashes)

for i in range(1, n + 1):  # De 1 a n
    print("#" * i)  # Imprime i hashes
